commands/logs.go

```diff
@@ -15,46 +16,74 @@ const maxNumberLogs = 1500
 
 // LogsCmd returns logs from an application on Section's delivery platform
 type LogsCmd struct {
-	AccountID       int    `required short:"a" help:"ID of account to query"`
-	AppID           int    `required short:"i" help:"ID of app to query"`
-	AppPath         string `default:"nodejs" help:"Path of NodeJS application in environment repository."`
-	InstanceName    string `default:"" help:"Specific instance of NodeJS application running on Section platform."`
-	Length          int    `default:100 help:"Number of log lines to fetch."`
+	AccountID    int    `required short:"a" help:"ID of account to query"`
+	AppID        int    `required short:"i" help:"ID of app to query"`
+	AppPath      string `default:"nodejs" help:"Path of NodeJS application in environment repository."`
+	InstanceName string `default:"" help:"Specific instance of NodeJS application running on Section platform."`
+	Length       int    `default:100 help:"Number of log lines to fetch."`
+	Follow       bool   `help:"Displays recent logs and leaves the session open for logs to stream in. --instance-name required."`
 	// StartTimestamp  int    `default:0 help:"Start of log time stamp to fetch."`
 	// EndTimestamp    int    `default:0 help:"End of log time stamp to fetch."`
 }
 
 // Run executes the command
 func (c *LogsCmd) Run() (err error) {
 	s := NewSpinner("Getting logs from app")
+	logsHeader := "\nInstanceName[Log Type]\t\t\tLog Message\n"

```

Comment:
Probably for a separate PR but it seems to me that if the user specified an exact instance name in the args, its redundant to repeat that instance name back to the user on every single log line emitted. 🤷

Reply:
I believe the intent is to keep the formatting consistent with when the instance name is not used. Heroku being the inspiration.